oaknational/Oak-Web-Application
# Feat/download card errors LESQ-406

## Description

- Adds new red colour to themes
- Updates formErrorMessages function
- Adds Icon to FieldError component and updates red
- Updates copy on messaging and test file
- Move position of error messages to above the section it is referring too (apart from list)

## AC:
- [x]  Add warning icon
- [x]  Move errors above the input fields
- [x]  Update error copy to be more descriptive (see Figma)
- [x]  Change the value of red colour from `failure: "#e51d4d"` to `red: "#DD0035"` (the new red is AAA for larger text and AA for smaller text)

## Issue(s)

Fixes #LESQ-406

## How to test

1. Go to https://deploy-preview-2054--oak-web-application.netlify.thenational.academy/teachers/programmes/combined-science-secondary-ks4-foundation-aqa/units/measuring-waves/lessons/transverse-waves/downloads
2. Clear local storage (or edit previously submitted details) if not done already to see the full downloads form
3. Deselect all, put in a fake email, don't enter a school name, don't accept terms and conditions to see all error messages
4. Click download to see bullet point list of error messages 

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-10-31 at 08 00 35](https://github.com/oaknational/Oak-Web-Application/assets/91190841/59443bd3-49d9-4d19-a403-af9ac562937a)
![Screenshot 2023-10-31 at 08 00 13](https://github.com/oaknational/Oak-Web-Application/assets/91190841/4571acc8-ad83-4675-9b25-90f9a65afe25)

How it should now look:
![Screenshot 2023-10-31 at 08 10 16](https://github.com/oaknational/Oak-Web-Application/assets/91190841/1359efa1-cb73-4860-8605-3e187c748517)
![Screenshot 2023-10-31 at 10 13 05](https://github.com/oaknational/Oak-Web-Application/assets/91190841/5801a4ab-f4ca-40a1-9d3b-799aa0ac40b5)




## Checklist
